languages/erlang/exercises/concept/basics/.docs/hints.md

```diff
@@ -0,0 +1,3 @@
+- You can use [arithmetic operators](http://erlang.org/doc/reference_manual/expressions.html#arithmetic-expressions) to implement the functions
+- Remember that the last expression _evaluated_ in the function body will be returned
+- Remember that [variables](http://erlang.org/doc/reference_manual/expressions.html#variables) must start with an uppercase letter
```

Comment:
*YES*! This is a good one, and fools me every now and then, when I do some Erlang after a streak of Python, Elixir or Go…